src/main/java/com/puppycrawl/tools/checkstyle/checks/design/HideUtilityClassConstructorCheck.java

```diff
@@ -74,6 +90,15 @@ public class HideUtilityClassConstructorCheck extends AbstractCheck {
      */
     public static final String MSG_KEY = "hide.utility.class";
 
+    /**
+     * Ignore classes annotated with the specified annotation(s). Name in
+     * this property must be an exact match for the annotation name on the class.
+     * If target class has annotation in fully qualified name (with package), this
+     * property should have such name too. If target class has annotation in simple
+     * name, this property needs to have the same simple name.
+     */
```

Comment:
This will need to be updated to reflect the changes above.

Reply:
done